docs/man/docker-events.1.md

```diff
@@ -76,7 +76,10 @@ relative to the current time on the client machine:
     2015-05-12T15:53:45.999999999Z07:00  7805c1d35632: (from redis:2.8) die
     2015-05-12T15:54:03.999999999Z07:00  7805c1d35632: (from redis:2.8) stop
 
+If `--since` is not provided, only new/live events be shown
```

Comment:
If the `--since` option is not provided, only new/live events will be show.

(looking for better wording, but don't have anything right now)